src/index.html

```diff
@@ -7,8 +7,65 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous | use-credentials"
+      >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;0,500;0,700;1,300;1,400;1,500;1,700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header_container">
+      <a href="#" class="logo_link">
+        <img
+          alt="logo"
+          src="./images/logo.png"
+          class="logo"
+        >
+      </a>
+
+      <nav class="nav_container">
+        <ul class="nav_item_list">
+          <li class="nav_item">
+            <a class="nav_item_link is-active" href="#">Apple</a>
+          </li>
```

Comment:
Let's fix classes everywhere
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a class="nav__link is-active" href="#">Apple</a>
          </li>
```